### DISCUSSION TITLE
[Feature] Add official support for caprover free and open source PasS

### DISCUSSION THREAD

**Discussion by gabrielgradinaru** [immich-app/immich#5341] [Feature Request]
### The feature

Caprover (https://caprover.com) is a free and open source PaaS that has multiple one click apps (https://caprover.com/docs/one-click-apps.html).

And immch is added as a one click app: https://github.com/caprover/one-click-apps/blob/master/public/v4/apps/immich.yml
The problem is that it is not official support and it is an old version.

Would you consider officially supporting the one click app from caprover?

With official support, users would be able to easily deploy immch on caprover and have confidence it will work as expected.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by burak-sayici** [comment]
I agree.
